tests/node_converters/gather_test.py

```diff
@@ -47,7 +53,11 @@ def test_gather() -> None:
         ],
         dtype=np.int64,
     )
-    _test_gather(input_array=input_tensor, indices=indices, axis=0, opset_version=9)
-    _test_gather(input_array=input_tensor, indices=indices, axis=1, opset_version=9)
-    _test_gather(input_array=input_tensor, indices=indices, axis=0, opset_version=13)
-    _test_gather(input_array=input_tensor, indices=indices, axis=1, opset_version=13)
+
+    all_variants = product(op_type_variants, axis_variants)
```

Comment:
in this case `@pytest.mark.parametrize` would be more compact and clear solution